sync_events.py

```diff
@@ -0,0 +1,222 @@
+import os
+import pytz
+import glob
+from dataclasses import dataclass
+from pathlib import Path
+
+import requests
+import yaml
+from slugify import slugify
+from dateutil.parser import parse as parse_date
+
+
+@dataclass
+class Event:
+    filename: str
+    pretix_slug: str
+    is_deleted: bool
+    title: str
+    event_date: str
+    event_time: str
+    event_registration: str
+    correlaidx: bool
+    languages: list[str]
+    tags: list[str]
+    description: str
+
+    @classmethod
+    def create(cls, api_event):
+        if api_event.get("slug") is not None:
+            # event
+            pretix_slug = api_event["slug"]
+        else:
+            # subevent
+            pretix_slug = f"{api_event['event']}/{api_event['id']}"
+
+        event = cls(
+            filename=slugify(cls._parse_title(api_event), max_length=50),
+            pretix_slug=pretix_slug,
+            is_deleted=False,
+            title=cls._parse_title(api_event),
+            event_date=cls._parse_date(api_event),
+            event_time=cls._parse_time(api_event),
+            event_registration="https://pretix.eu/correlaid/" + pretix_slug,
+            correlaidx=False,
+            languages=[],
+            tags=[],
```

Comment:
* correlaidx
* languages
* tags

these values are not provided by the pretix API (i think), so i put some not-super-useful default values here. i wonder if there is a way to improve those values / anywhere else to get that information.

Reply:
re languages: 
- if pretix english and german -> english
- if  pretix only english -> english
- if pretix only german -> german

Reply:
tags: can be added manually

Reply:
added a commit for the languages now.